README.md

```diff
@@ -1,28 +1,153 @@
-# discovery
+# Discovery Service
 
-## Build
+## Main principles and features described
+### Features
+* Fetch MYST prices from the public API
```

Comment:
Maybe thats just me but I would argue this is neither a feature nor a principle this is just a job we have to do to calculate things further down the line 🤷🏻

Reply:
technicaly it's a feature, but it's not strict. Just add this bullets to give understanding what it do